### DISCUSSION TITLE
deprecating my snap building repository ask for a fork

### DISCUSSION THREAD

**Discussion by Aury88** [Orama-Interactive/Pixelorama#410] [General]
I will no longer maintain my account on github so all my repositories, included the one used to obtain the snapped version for pixelorama, will be removed.
 I'm asking here if anyone is willing to maintain a fork. You can send me your account username in snapcraft.io to https://forum.snapcraft.io/u/Aury88 and I will coordinate with the snapcraft team to give you the "ownership" of the app name in the snapstore. probably they will ask you your registration email. 

The working repository is here: https://github.com/Aury88/pixelorama
the process used by me to compile is really easy:

before making any changes, remove the pixelorama folder on your desk.
from terminal in the directory where you want the new folder (for example ~/project)
`git clone https://github.com/your_name/pixelorama/`
  remove unnecessary/old "files" folder with
```
git rm -r files`
git commit. -m "Commit description"`
git push origin master`
```
 download from  https://orama-interactive.itch.io/pixelorama/ the linux version for amd64
 extract the file, rename in "files" and paste through file manager in the root directory (for example ~/project/pixelorama)
```
git add filename / filename
git commit. -m "Commit description"
git push origin master
```
  in github now you can make the version change in the snapcraft.yaml (for example from `version: '0.8.2'` to `version: '0.8.3'`)
the building process will be done from snapcraft automatically.

**Comment by OverloadedOrama** [comment]
Hello. I could try and maintain it, the process seems rather straightforward. [I made a snapcraft.io account, and my username is OverloadedOrama](https://forum.snapcraft.io/u/overloadedorama/). I'm new to it and I don't see a way to message you there though.

I have a question though, is it necessary to remove all files, commit, add the new files and then commit again? Won't it work if we just commit once, or by simply replacing (without deleting) the old files with the new ones? (Except from when there have been changes in the pixelorama_data, of course). It would also be interesting if I attempted to automate the process with GitHub Actions.

**Comment by OverloadedOrama** [comment]
I forked the snap repo, but because I already have a repository named "Pixelorama", it got renamed to "pixelorama-1". I'd like to change it to something like "pixelorama-snap", will this work?

**Reply by Aury88** [reply]
Hi @OverloadedOrama.
I'm very glad to see you are interested. I [posted on the forum ](https://forum.snapcraft.io/t/pixelorama-publisher-transition/21902) a request and and I'm waiting instructions.
I'm very new to git/github and only know few commands, so probably my method is not the smartest. The only thing important is to have the "files" folder updated on github with the latest version you can find in the downloadable zip for linux on AMD64. The method used to obtain this is probably not important, so probably [you can also automate  the process with GitHub Actions](https://snapcraft.io/tutorials/continuous-snap-delivery-from-circle-ci#1-overview).
Also the repo name is not important. the snap name is based on the registration name in the  snap store and the name declared in the snapcraft.yaml file. You have only to update the [targeting github repository on snapcraft for the building process](https://snapcraft.io/build)

Thank you again.
